### PR TITLE
feat/colors-plugin-option

### LINKED ISSUE #5: Make an option to make it more visible on white backgrounds

### PR DESCRIPTION
**Pull request by PaulieScanlon**
- add colors plugin option
- update README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,19 @@ module.exports = {
...
 
+## Options
+
+You can pass an array of colors to use via the plugin options
+
+```
+  {
+    resolve: "@raae/gatsby-plugin-let-it-snow",
+    options: {
+      colors: ["#fff000", "#ff00ff", "#00ff00"],
```
Comment (raae) [inline README.md:33]: **praise:** I like this solution a lot 🎉

**Review by raae — CHANGES_REQUESTED** [review]
**suggestion (non-blocking):** add `pluginOptionsSchema` to validate the options
**chore:** rebase

File: README.md [hunk 2]
```diff
@@ -46,6 +59,8 @@ To get your changes merged into the project create a Pull Request from your fork
 
 ## How to contribute in other ways
 
+> > > > > > > 54b548bee5ce7298511a8cd46ffe8cde891a4343
```
Comment (raae) [inline README.md]: **chore:** Forgot to remove this line it seems

File: plugin/gatsby-node.js [hunk 3]
```diff
@@ -1 +1,8 @@
 // https://www.gatsbyjs.com/docs/reference/config-files/gatsby-node/
+export const pluginOptionsSchema = ({ Joi }) => {
+  return Joi.object({
+    colors: Joi.array()
+      .items(Joi.string())
```
Comment (raae) [inline plugin/gatsby-node.js]: **nitpick:** You can add a default value here

**Review by raae — CHANGES_REQUESTED** [review]

File: plugin/gatsby-node.js [hunk 4]
```diff
@@ -1 +1,8 @@
 // https://www.gatsbyjs.com/docs/reference/config-files/gatsby-node/
+export const pluginOptionsSchema = ({ Joi }) => {
```
Comment (raae) [inline plugin/gatsby-node.js]: **chore:** Needs to be exports

**Review by raae — CHANGES_REQUESTED** [review]
If you rebase from main now you can do yarn install, and get proper environment.

File: plugin/gatsby-node.js (hunk 4 above)
Comment (PaulieScanlon) [reply plugin/gatsby-node.js]: bums... ok changed it to `exports.pluginOptionsSchema`

**Pull request merged**